Modix.Bot/ModixBot.cs

```diff
@@ -145,8 +151,9 @@ void OnStopping()
 
                 _client.Log -= _serilogAdapter.HandleLog;
                 _commands.Log -= _serilogAdapter.HandleLog;
-
                 _restClient.Log -= _serilogAdapter.HandleLog;
+
+                _commands.CommandExecuted -= HandleCommandResultAsync;
```

Comment:
Should any remaining command scopes be disposed of here?

Reply:
Good idea